11-payment-encoding.md

```diff
@@ -140,6 +141,9 @@ Currently defined tagged fields are:
    * `fee_base_msat` (32 bits, big-endian)
    * `fee_proportional_millionths` (32 bits, big-endian)
    * `cltv_expiry_delta` (16 bits, big-endian)
+* `9` (5): `data_length` variable. One or more bytes containing features
```

Comment:
Not related to that change, but it seems like Bolt 11 rolled out a custom TLV before TLV was cool.
Should we update Bolt 11 to use a TLV stream?

Reply:
Yeah, if we were doing it again we probably would.  But bolt11 is natively 5-bit units, so it's a little boutique.

Reply:
Ok we save more space than TLV by using only 5 bits, understood.
We can discuss that later to see if it's worth changing.